contributing/code/standards.rst

```diff
@@ -234,14 +248,17 @@ Documentation
 
 * Omit the ``@return`` tag if the method does not return anything;
 
-* The ``@package`` and ``@subpackage`` annotations are not used.
+* The ``@package`` and ``@subpackage`` annotations are not used;
+
+* Inline the ``@inheritdoc`` tag.
 
 License
--------
+~~~~~~~
 
 * Symfony is released under the MIT license, and the license block has to be
   present at the top of every PHP file, before the namespace.
 
+.. _`PHP CS Fixer tool`: https://github.com/FriendsOfPHP/PHP-CS-Fixer
```

Comment:
website is up to date.
- it provides information about newest v2.4 release
- copyrights you have mentions is for logo only, not website, probably we could bump it, why not...
- logo was dedicated to be used for PHP CS Fixer and I did agreed with Fabien to have it be dedicated for PHP CS Fixer now
- it was not moved to SensioCloud but github.io page to let community fully control it (eg trigger web updates), check the CNAME of domain. I do know that old s3 bucket still contains old data, but it's not in use at all. Similar, `get.sensiolabs.org` could stop providing PHP CS Fixer (maybe it already stopped, not sure, asked Fabien months ago)

I think you have quite outdated informations.